views/lib/forms/form-auto-logout-alert.hbs

```diff
@@ -0,0 +1,48 @@
+<style>
+    .sloth {
+        width: 35%; 
+        margin-bottom: -50px;
+        height: auto; 
+        opacity: 0.6;
+    }
+
+    .sloth-text {
+        text-align: center;
+        padding: 15px;
+    }
+
+    @media (min-width: 575.98px){
+    .sloth {
+        float: right;
+        margin-top: -12px;
+        margin-bottom: -20px;
+        text-align: center;
+    }
+
+    .sloth-text {
+        text-align: left;
+        padding: 35px 0 10px 28px;
+    }
+    }
+</style>
+<div class="form-group" data-show-on-remaining-sec="{{#global "JWT_SHOW_TIMEOUT_WARNING_SECONDS"}}{{/global}}">
```

Comment:
as mentioned before, this is no if else like structure, you may just register globals in root scope for handlebars